tests/initrd.c

```diff
@@ -4,6 +4,20 @@
 #include <mount.h>
 #include <vm_map.h>
 #include <ktest.h>
+#include <dirent.h>
+
+static void prepare_uio(uio_t *uio, iovec_t *iov, char *buffer, int buf_size) {
```

Comment:
I have added prepare_uio function in another PR. After that PR gets merged with master I'll update this branch.